src/main/java/no/nav/pto/veilarbportefolje/arenapakafka/aktiviteter/TiltakRepositoryV2.java

```diff
@@ -77,6 +87,23 @@ kode IN (SELECT DISTINCT tiltakskode FROM brukertiltak BT
         );
     }
 
+    public void leggTilTiltak(String aktoerIder, HashMap<AktorId, List<AktivitetEntityDto>> result) {
+        namedDb.query("""
+                        SELECT aktoerid, tildato, fradato, tiltakskode FROM brukertiltak
+                        WHERE aktoerid = ANY (:ids::varchar[])
+                        """,
+                new MapSqlParameterSource("ids", aktoerIder),
+                (ResultSet rs) -> {
+                    while (rs.next()) {
+                        AktorId aktoerId = AktorId.of(rs.getString("aktoerid"));
+                        AktivitetEntityDto aktivitet = mapTiltakTilEntity(rs);
+
+                        leggTilAktivitetPaResultat(aktoerId, aktivitet, result);
```

Comment:
Why this function can't return result, and then parent class/method can collect all different results and do what is necessary with them.